apps/web/src/pages/templates/TemplatesListNoData.tsx

```diff
@@ -10,6 +10,8 @@ import { IBlueprintTemplate } from '../../api/types';
 import { TemplateCreationSourceEnum } from './shared';
 import { useHoverOverItem } from '../../hooks';
 import { EchoProjectCardTile } from './components/EchoProjectWaitList';
+import { Test } from '@novu/novui';
+import { css } from '@novu/novui/css';
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** ‏Example of new imports!